README.md

```diff
@@ -106,50 +109,54 @@ To run everything, run:
 make check
 ```
 
-## Comparison of developer environments
+## Environments
 
-For end-to-end client testing on Qubes, make sure you either run the packaged client or run the unpackaged client with proxy turned on.
+The quickest way to get started with running the client is to use the [developer environment](#developer-environment) that [runs against a test server running in a local docker container](#running-against-a-test-server). This differs from a staging or production environment where the client receives and sends requests over Tor. Things are a lot snappier in the developer environment and can sometimes lead to a much different user experience, which is why it is important to do end-to-end testing in Qubes using the [staging environment](#staging-environment), especially if you are modifying code paths involving how we handle server requests and responses.
 
-### Developer environment on non-Qubes
+For reproducing production bugs or running demos, we recommend using the [Production Environment](#production-envrionment) that will allow you to test a nightly build of the client.
 
-* Ran by `run.sh`
-* Uses temporary configuration directories by default
-* Works with SecureDrop running in a local docker container, see [SecureDrop docs](https://docs.securedrop.org/en/latest/development/setup_development.html) for setup instructions
-* Tor is not used
-* Does not support opening submissions
-* Does not use `split-gpg`, instead uses a development gpg private key inside a gpg keychain stored in the temporary configuration directories
+We support running the [developer environment on a non-Qubes OS](#developer-environment-on-a-non-qubes-os) for developer convenience. If this is your preferred environment, keep in mind that you, or a PR reviewer, will need to run tests in Qubes if you modify code paths involving any of the following:
 
-### Developer environment on Qubes (no-proxy)
+* cryptography
+* opening of files in VMs
+* network (via the RPC service) traffic
+* fine tuning of the graphical user interface
 
-* Ran by `run.sh`
+### Developer environment
+
+* Ran by `run.sh` inside a virtual env on the `sd-dev` AppVM
```

Comment:
Grammar nit, would you mind changing these "Ran by"s to "Run by"s while you're here? :)